draft-ietf-quic-qpack.md

```diff
@@ -103,36 +104,47 @@ when, and only when, they appear in all capitals, as shown here.
 
 Definitions of terms that are used in this document:
 
-Header field:
+HTTP field line:
 
-: A name-value pair sent as part of an HTTP message.
+: A name-value pair sent as part of an HTTP field section.  See Section 4 of
+  {{!SEMANTICS=I-D.ietf-httpbis-semantics}}.
 
-Header list:
+HTTP field value:
 
-: An ordered collection of header fields associated with an HTTP message.  A
-  header list can contain multiple header fields with the same name.  It can
-  also contain duplicate header fields.
+: Data associated with a field name, composed from all field line values with
+  that field name in that section, concatenated together and separated with
+  commas.
 
-Header block:
+Field section:
 
-: The compressed representation of a header list.
+: An ordered collection of HTTP field lines associated with an HTTP message.  A
+  field section can contain multiple field lines with the same name.  It can
+  also contain duplicate field lines.  An HTTP message can include both header
+  field and trailer field sections.
+
+Representation:
+
+: An instruction which represents a field line, possibly by reference to the
+  dynamic and static tables.
 
 Encoder:
 
-: An implementation which transforms a header list into a header block.
+: An implementation which transforms a field section into a series of
```

Comment:
"a field section" should probably be plural, because encoders and decoders typically transform multiple field sections.